main.go

```diff
@@ -367,7 +367,9 @@ func Test(pkgName, target string, config *BuildConfig) error {
 		if err != nil {
 			// Propagate the exit code
 			if err, ok := err.(*exec.ExitError); ok {
-				os.Exit(err.ExitCode())
+				if status, ok := err.Sys().(syscall.WaitStatus); ok {
+					os.Exit(status.ExitStatus())
+				}
```

Comment:
I think this is missing a case where the error is not `syscall.WaitStatus`.
```suggestion
				}
				os.Exit(1)
```

Reply:
OK, change made.